catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/RegistryConstants.java

```diff
@@ -16,6 +16,8 @@
 public class RegistryConstants {
     public static final String REGISTRY_TAG = "registry";
 
+    public static final String REGISTRY_TAG_INTERNAL = "registry.internal";
```

Comment:
Not sure what I think about periods in tags and the use of internal for this tag since I assume metacards in it will be shared externally.  Maybe `registry-local` or you use `registry` tag for local items and put remote items in `registry-remote`.